docs/faq.rst

```diff
@@ -987,6 +987,12 @@ stable. If the directory where you mount a filesystem is different every time,
 Borg assumes they are different files. This is true even if you backup these
 files with relative pathnames - borg uses full pathnames in files cache regardless.
 
+It is possible for some filesystems, such as mergerfs_,
+to return inconsistent inode numbers across runs, causing borg to consider them
+changed. A workaround is to set the option ``--ignore-inode`` so that files with
+different inodes won't be treated as modified.
+
+.. _mergerfs: https://github.com/trapexit/mergerfs
```

Comment:
i don't think we need to link to mergerfs.

Reply:
I've removed the link.